src/app/zones/locales/en-us/index.yaml

```
@@ -131,7 +131,18 @@ zones:
         connectDescription: 'Next, run the following command to connect the Zone.'
         connectCommand: 'helm install -n kuma-system -f values.yaml kuma kuma/kuma'
   create:
-    errorTitle: 'Could not create the Zone'
+    generalError:
+      title: 'Could not create the Zone'
+    statusError:
+      400:
+        title: 'The Zone name {zoneName} is invalid'
+        description: 'Zone names may contain alphanumerical characters, dashes (-), and underscores (_).'
+      409:
+        title: 'A Zone with the name {zoneName} already exists'
+        description: 'If you want to connect a Zone with this name, you can delete the existing one and recreate it.'
```

If I'd really have to be nitpicky then I'd change it like this:

```suggestion
        description: 'If you want to connect a Zone with this name, you can delete the existing one and create a new one.'
```